fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/networking/ClientEntrypoint.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.networking;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.event.network.S2CPacketTypeCallback;
+import net.fabricmc.fabric.api.networking.v1.event.ClientPacketChannelCallback;
+
+public final class ClientEntrypoint implements ClientModInitializer {
```

Comment:
Can there be a better name?

Reply:
NetworkingClient?

Reply:
Won't change the name. This isn't public api and this networking v0 is getting deprecated.